src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -297,6 +297,10 @@ private async Task<IDictionary<string, string>> ValidateFunctionAppPublish(Site
                             ColoredConsole.WriteLine(WarningColor($"Setting '{Constants.FunctionsWorkerRuntime}' to 'dotnet-isolated'"));
                             result[Constants.FunctionsWorkerRuntime] = WorkerRuntimeLanguageHelper.GetRuntimeMoniker(workerRuntime);
                         }
+                        else if (workerRuntime == WorkerRuntime.None)
+                        {
+                            throw new ArgumentException($"Your local project is set to '{workerRuntime}' is not a valid option. Options are {WorkerRuntimeLanguageHelper.GetAvailableOptions()}");
```

Comment:
```suggestion
                            throw new ArgumentException($"Your local project is set to '{workerRuntime}', this is not a valid option. Available options are: '{WorkerRuntimeLanguageHelper.GetAvailableOptions()}'");
```

Also what does this `WorkerRuntimeLanguageHelper.GetAvailableOptions()` actually look like in the CLI when it occurs? Can you share a screenshot?

Reply:
Hi @liliankasem , the method mentioned above has been removed, as it is not needed, since we already have a similar method, which I missed earlier.